spec/models/drink_spec.rb

```diff
@@ -1,6 +1,23 @@
-require "spec_helper"
+require "rails_helper"
 
 describe Drink do
   it { should have_many(:reviews).dependent(:destroy) }
   it { should have_many(:drink_liquors).dependent(:destroy) }
+
+  context 'searching' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.